content/en/blog/falcosidekick-2-27-0-ui-2-1-0/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Falcosidekick 2.27.0 and Falcosidekick-UI 2.1.0
+linktitle: Falcosidekick 2.27.0 and Falcosidekick-UI 2.1.0
+date: 2022-12-16
+author: Thomas Labarussias
+slug: falcosidekick-2-27-0-ui-2-1-0
+images:
+  - images/falcosidekick-featured.png
+---
+
+![](images/falcosidekick-featured.png)
+
+So much good things happened for Falcosidekick and Falcosidekick UI this year, it's still incredible such projects became so beloved and useful for the community. To all contributors, promotors and users, a big big thank you.
```

Comment:
So many good things happened for Falcosidekick and Falcosidekick UI this year. It's still incredible these projects became so beloved and useful for the community. To all contributors, promotors and users, a big, big thank you.